test/posix-mock-test.cc

```diff
@@ -6,7 +6,7 @@
 // For the license information refer to format.h.
 
 // Disable bogus MSVC warnings.
-#ifndef _CRT_SECURE_NO_WARNINGS
+#if !defined(_CRT_SECURE_NO_WARNINGS) && defined(_MSC_VER)
```

Comment:
Why check if _MSC_VER is defined?

Reply:
As per commit message: to fix the -Wreserved-identifier warnings.  Underscore + uppercase is reserved, and #defineing such a name causes a warning.  No need to define in on unix since it won't do anything, thus the warning is avoided.

Reply:
But it's not an identifier. What compiler gives this warning?

Reply:
clang.

```
% cat test.cxx
#define _FOO

int main (void)
{
	return 0;
}


% clang -fsyntax-only -Weverything ~/Desktop/test.cxx 
/Users/sean/test.cxx:1:9: warning: macro name is a reserved identifier [-Wreserved-macro-identifier]
#define _FOO
        ^
```